docs/source/Usage.md

```diff
@@ -235,6 +235,9 @@ The following is the list of the available analyzers you can run out-of-the-box.
 * `DNSDB`: scan an observable against the [Passive DNS Farsight Database](https://www.farsightsecurity.com/solutions/dnsdb/) (support both v1 and v2 versions)
 * `DNS0_EU`: Retrieve current domain resolution with DNS0.eu DoH (DNS over HTTPS)
 * `DNS0_EU_Malicious_Detector`: Check if a domain or an url is marked as malicious in DNS0.eu database ([Zero](https://www.dns0.eu/zero) service)
+* `DNS0_names`: Run advanced searches on billions of current and historical domain names.
+* `DNS0_rrsets_data`: Query billions of current and historical DNS resource records sets. Performs right-hand side matching.
+* `DNS0_rrsets_name`: Query billions of current and historical DNS resource records sets. Performs left-hand side matching.
```

Comment:
please add a reference to the DNS0 service by mentioning it and by adding a link